### PR TITLE
Replace matrix links with zulip links

### PR DESCRIPTION
**Pull request by kathatherine**
### Description

Closes https://github.com/conda/conda-dot-org/issues/230.

We're moving away from Matrix and towards Zulip, so I've updated all relevant links to their Zulip equivalents.

- ~[ ] If I have added a new page to `learn/` or `community/`, I have added it to the corresponding `_sidebar.json` file.~

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,9 +4,9 @@
 
 This is the repository for the conda.org community website. To become involved:
 
-- Join the [conda.org chat room](https://app.element.io/#/room/#conda.org:matrix.org), which is part of the [conda space on Matrix](https://app.element.io/#/room/#conda:matrix.org)
+- Join the [conda-dot-org channel](https://conda.zulipchat.com/#narrow/channel/471465-conda-dot-org), which is part of the [conda organization on Zulip(https://conda.zulipchat.com)
 - Visit the [Root HackMD page](https://hackmd.io/DGtozSlsSjSokpYAK5-9hw), which links to everything in HackMD.
-- We meet every two weeks on Monday at 9am US Central time / 16:00 Central European Time. Please join the [Matrix room](https://app.element.io/#/room/#conda.org:matrix.org) and we will invite you.
+- We meet every two weeks on Monday at 9am US Central time / 16:00 Central European Time. Please join the [Zulip channel](https://conda.zulipchat.com/#narrow/channel/471465-conda-dot-org) and we will invite you.
```
Comment (jaimergp) [inline README.md:9]: I think we can remove this, since we haven't meet for a while now.

Comment (jaimergp) [reply README.md:9]:
```suggestion
```

Comment (kathatherine) [reply README.md:9]: Good point, Jaime

**Review by jaimergp — APPROVED** [review]
This is looking good! I was looking for more Matrix/Element occurrences in the website and found [this blog](https://github.com/conda/conda-dot-org/blob/52c5a212586eca376eb122233f5acfd02c09c94e/blog/2023-02-01-matrix.mdx#L2)... do you think we should write a similar post? Maybe in collaboration with @dashagurova? Doesn't have to be in this PR, of course.

**Comment by kathatherine** [comment]
Probably a good idea to advertise it, yes! It doesn't have to be as long as the Matrix post Dave wrote, I don't think.

**Comment by jaimergp** [comment]
Let's merge here and start a new PR with a simple draft.

**Pull request merged**